### ISSUE TITLE
Add theme switch button for light/dark mode

### ISSUE THREAD

**Issue by batradilshan18** [anurag2787/DateNow#45]
Hi, I am a GSSoC'25 Contributor.
The website does not have a theme switch button to toggle between light and dark modes. I will add this functionality for better accessibility.
I would like to work on this alone.
Kindly assign this issue to me.
@anurag2787
